tests/scenarios/helpers/command-watcher.ts

```diff
@@ -108,7 +108,7 @@ export default class CommandWatcher {
       return;
     }
 
-    this.process.kill();
```

Comment:
vite needs to be killed with `SIGINT`, otherwise it leaves the server subprocess running